HelpSource/scdoc.js

```diff
@@ -190,6 +190,39 @@ function fixTOC() {
         });
     });
 
+    // if served via web, show theme switcher
+    // within IDE use the theme settings of the IDE to control the docs theme
+    if(window.location.protocol.startsWith("http")) {
```

Comment:
That means that helpfile opened locally in a help browser doesn't have this selector, and it won't follow the IDE theme (unless it would be possible to generated the css file dynamically when the IDE changes theme). That's not the end of the world, but maybe there would be a way to hide the selector only in the HelpBrowser opened by IDE, but show elsewhere, even when serving a local file?